### PR TITLE
Add small missing info to the readme

### PR DESCRIPTION
**Pull request by vladimirsouza**
I realized that there is no need to edit `gambl/config.yml` file. We just set our `R` working directory to `GAMBLR` or `gambl_results`, which contain up-to-date `config.yml` files. Then, I remade this PR entirely and also added some new changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ Sys.setenv(R_CONFIG_ACTIVE = "remote")
 check_gamblr_config()
 ```
 
-15. You should now be all set to explore a collection of GAMBLR function remotely on your local machine. For example you could try the following test code to ensure your setup was successful. For a set of comprehensive examples and tutorials, please refer to the [test_remote.r](https://github.com/morinlab/GAMBLR/blob/master/test_remote.R) script.
+15. You should now be all set to explore a collection of GAMBLR function remotely on your local machine. For example you could try the following test code to ensure your setup was successful. For a set of comprehensive examples and tutorials, please refer to the [test_remote.r](https://github.com/morinlab/GAMBLR/blob/master/resources/test_remote.R) script.
```
Comment (mattssca) [inline README.md:183]: Nice catch!

**Review by Kdreval — APPROVED** [review]

**Comment by vladimirsouza** [comment]
I had to `rebase` this branch because other commits have been pushed to master after this branch was created.

**Comment by vladimirsouza** [comment]
Sorry for the extra work. Now I think the rebase wasn't that important.

**Comment by Kdreval** [comment]
No worries! Unless there are merge conflicts (which will show up here on the PR page) the merge would happen automatically and you don't have to re-base. I'll wait till the actions will complete for the latest push and will merge this update then 👍

**Review by rdmorin — APPROVED** [review]

**Pull request merged**